### DISCUSSION TITLE
DB2CB6F5-247F-761C-4001-072A6C6BE5FD

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#51] [General]
# DB2CB6F5-247F-761C-4001-072A6C6BE5FD

The International Space Station (ISS) is a multinational collaborative project between five space agencies around the world (NASA, Roscosmos, ESA, JAXA, CSA), which serves as a microgravity and space environment research laboratory for astrobiology, astronomy, meteorology, physics, and other fields. The ISS is suited for testing the spacecraft systems and equipment required for possible future long-duration missions to the Moon and Mars.

https://celestia.mobi/addon?item=DB2CB6F5-247F-761C-4001-072A6C6BE5FD&lang=zh_CN

**Comment by 13940951572** [comment]
空间站的位置和现在不一样！

**Reply by DaveBowman2001** [reply]
The ISS's orbit in Celestia does not (and **will not ever**) reflect its actual position in real life. This is due to various reasons:
- The station regularly changes its orbit to maintain a roughly 400 km altitude above Earth's surface, avoid space debris encounters, and/or receive incoming visiting crew and cargo spacecraft. 
- The natural orbital decay is beyond the capabilities of Celestia's own programming, thus the simulated positions already deviate from actual data in just over a few seconds
- Although there is available orbital data over CelesTrak and JPL HORIZONS, the resulting data is simply too large to accurately track the station since its launch in 1998

**Reply by DaveBowman2001** [reply]
That being said, Linfeng Li produced a small addon that *will* update the orbit of the ISS to its real life position on a given minute. It can be found [**here**](https://celestia.mobi/resources/item?item=94C88086-0F67-F172-8C14-957BEB20846D). 

Look for a small button named `Update ISS Position`. Take note that it still cannot track the station's actual position in real time due to reasons explained above

**Reply by galileokeplertycho** [reply]
I think the textures are too heavy., they crash my game, Please reduce their texturess size so that it works on all devices !!!!!!

**Reply by DaveBowman2001** [reply]
I'm deeply sorry, but the textures are the lowest I can get them. This addon is not intended for older devices, as the models alone (which I imported straight from [NASA's ISS model](https://www.google.com/url?sa=t&source=web&rct=j&url=https://solarsystem.nasa.gov/resources/2378/international-space-station-3d-model/&ved=2ahUKEwiJsvXCr5z_AhXc9DgGHfG4ATIQFnoECBMQAQ&usg=AOvVaw1ZlHMqPbjy62e3MsJG0aP3)) requires a lot of memory to render

**Comment by 13940951572** [comment]
更新lss小按钮在哪里？

**Reply by DaveBowman2001** [reply]
It's on the addon link I've provided on my reply above. It should redirect you to an addon named "`ISS (Dynamic)`", the `Update ISS Position` button is located there

**Reply by 13940951572** [reply]
我点击了怎么老是显示无效文件格式而且也没反应？
